cluster-assign-roles.sh

```diff
@@ -47,7 +47,7 @@ function install_machines {
     else
       printf "About to bootstrap node $fqdn in $ENVIRONMENT run_list ${run_list}...\n"
       ./chefit.sh $ip $ENVIRONMENT
-      sudo -E knife bootstrap -E $ENVIRONMENT -r "$run_list" $ip -x ubuntu ${passwd:+-P} $passwd $KNIFE_ADMIN --sudo <<< $passwd
+      sudo -E knife bootstrap --node-ssl-verify-mode none --no-node-verify-api-cert -E $ENVIRONMENT -r "$run_list" $ip -x ubuntu ${passwd:+-P} $passwd $KNIFE_ADMIN --sudo <<< $passwd
```

Comment:
Regarding trusting the certs, this doc is super succinct and gives the procedure for importing them into each node:

http://jtimberman.housepub.org/blog/2014/12/11/chef-12-fix-untrusted-self-sign-certs/

Of course this doesn't actually make the problem of getting the server to push certs to the clients any easier to resolve in an automated fashion (chicken and egg anyone?)...